dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js

```diff
@@ -95,10 +96,15 @@ const findOneGefDeal = async (_id, callback) => {
   if (deal) {
     deal.facilities = await findAllGefFacilitiesByDealId(_id);
```

Comment:
Low priority - These calls could be done in parallel

Reply:
Good point, they'll be another call we need to add here so i'll do this in another PR